### DISCUSSION TITLE
part-ii/2015-78/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#197] [Question comments]
Part II 2015 Logic and Set Theory: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2015-78/

**Comment by ArthurConmy** [comment]
Bookwork, up to and including that the relation $<$ on some set $X$ is well-founded if all non-empty $S \subseteq X$ have an $<$-minimal element, some $s \in S$ so $t < s$ does not hold for any $t \in S$.

If $<$ extends to a well-ordering then $<$ must be well-founded as if not, we'd have some $ y_1 > y_2 > ... $ contradicting well-ordering.

$<$ well-founded does not imply every total ordering extending it is a well-ordering. Consider the set $\mathbb{ Q}_{>0}$ of positive rationals with relation $q_1 <' q_2$ iff $q_2 > q_1$ and $q_2 - q_1 \in \mathbb{ Z}$, which extends to the relation $<$ on $\mathbb{ Q}_{>0}$.
